README.md

```diff
@@ -109,6 +109,174 @@ Enable the rules that you would like to use.
   }
 ```
 
+### Shareable configs
+
+#### Recommended
+
+This plugin exports a `recommended` configuration that enforces React good practices.
+
+To enable this configuration use the `extends` property in your `.eslintrc` config file:
+
+```json
+{
+  "extends": ["eslint:recommended", "plugin:react/recommended"]
+}
+```
+
+See [`eslint` documentation](https://eslint.org/docs/user-guide/configuring/configuration-files#extending-configuration-files) for more information about extending configuration files.
+
+#### All
+
+This plugin also exports an `all` configuration that includes every available rule.
+This pairs well with the `eslint:all` rule.
+
+```json
+{
+  "plugins": [
+    "react"
+  ],
+  "extends": ["eslint:all", "plugin:react/all"]
+}
+```
+
+**Note**: These configurations will import `eslint-plugin-react` and enable JSX in [parser options](https://eslint.org/docs/user-guide/configuring/language-options#specifying-parser-options).
+
+## Configuration (new: `eslint.config.js`)
+
+From [`v8.21.0`](https://github.com/eslint/eslint/releases/tag/v8.21.0), eslint announced a new config system.
+In the new system, `.eslintrc*` is no longer used. `eslint.config.js` would be the default config file name.
+In eslint `v8`, the legacy system (`.eslintrc*`) would still be supported, while in eslint `v9`, only the new system would be supported.
```

Comment:
> while in eslint `v9`, only the new system would be supported

Where is this part coming from? ESLint has not announced the end-of-life for `.eslintrc`.

Reply:
whew, i'm really glad to hear that. let's remove this line unless there's a citation.

Reply:
This same phrase seems to be used in other community plugins as well, https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1886.

I just asked about this on ESLint chat and the maintainers said no such announcement about EOL has been made.

Reply:
That's the same contributor putting it in there, so it may come from the same misunderstanding.

I can't comment on that PR, so it'd probably be helpful if you commented there as well about the incorrect EOL info.

Reply:
@AriPerkkio Thanks for fixing my confusion again :)
(He also did the same in another repo.)